melodic/distribution.yaml

```diff
@@ -2071,6 +2071,24 @@ repositories:
       url: https://github.com/dingo-cpr/dingo.git
       version: master
     status: developed
+  dingo_simulator:
+    doc:
+      type: git
+      url: https://github.com/dingo-cpr/dingo_simulator.git
```

Comment:
The package.xml files in the source properly declare a license (good), but there is no LICENSE file there.  Could you please add one?  No need to do another release after that, just having it in the source repository is good enough for me.

Reply:
Added!

Reply:
Great, thanks.